site/class-2-style/bjorn2123/index.css

```diff
@@ -0,0 +1,113 @@
+html {
```

Comment:
Hey!!! You’ve changed my CSS!

This assignment is about refactoring code, not about changing it!

Reply:
(it does look beautiful, but it’s not the assignment... :/)